app/controllers/settings/followers_controller.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+class Settings::FollowersController < ApplicationController
+  layout 'admin'
+
+  before_action :authenticate_user!
+
+  def show
```

Comment:
Should the `@domains` here be paginated, maybe ordered by most followers? I'm not sure what's typical for how many instances a typical account has following them.

If so, maybe rename this action to `index`